src/components/AppContent/AppContent.vue

```diff
@@ -90,4 +147,47 @@ export default {
 	background-color: var(--color-main-background);
 }
 
+#app-content-wrapper {
+	overflow: hidden;
+	height: calc(100vh - var(--header-height));
+}
```

Comment:
Sorry, the `calc(100vh - var(--header-height));` trick is not acceptable. Some mobile browsers does not count the bottom touch bar in the `100vh` meaning it will still scroll vertically with the `100vh-headerheight`. You need to comply to the inherited height and the `display flex`.

Reply:
Good call. I've seen it used like that in `AppSidebar` and thought it was okay. It does need a specific height though, otherwise it'll mess up the individual scrolling of the sidebar and the main content. I went for a JS-calculated approach which mimics `calc(100vh - var(--header-height))` which looks like this: `window.innerHeight - document.getElementById('header').offsetHeight`. It is set on mount and updated an `window.resize` (new eventListener added). If there's a CSS-only solution I am missing here, please let me know!